src/ui/atoms/ContentToggle.test.tsx

```diff
@@ -65,4 +60,32 @@ describe('ContentToggle', (): void => {
         );
         expect(container.querySelector('.select-field')).toBeInTheDocument();
     });
+
+    it('should open ContentToggle and detect there is content', (): void => {
+        const { container } = render(
+            <ContentToggle
+                id="test"
+                openText="Remove secondary country of residence/affiliation"
+                collapsedText="Add secondary country of residence/affiliation"
+            >
+                <SelectField id="test" labelText="This is label text" values={[]}></SelectField>
+            </ContentToggle>,
+        );
+
+        fireEvent.click(container.querySelector('.content-toggle__toggle-btn'));
+        expect(container.querySelector('.select-field')).toBeInTheDocument();
+    });
+
+    it('should open ContentToggle and detect there is no content', (): void => {
```

Comment:
Have a re-read of the 2 test cases that @Cooryd asked for. You've covered one of the tests, but this second test isn't correct and should be testing a different user interaction. 

As an aside, this second test here isn't actually testing anything and there is no way it would ever fail.

Reply:
The second test when I added the Selectfield in with the toBeNull it failed but passed when I took it out.